src/app/pages/apps/components/pool-configuration-checker/pool-configuration-checker.component.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  AfterViewInit, ChangeDetectionStrategy, Component, EventEmitter, Output,
+} from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { ApplicationsService } from 'app/pages/apps-old/applications.service';
+import { SelectPoolDialogComponent } from 'app/pages/apps/components/select-pool-dialog/select-pool-dialog.component';
+
+@UntilDestroy()
+@Component({
+  selector: 'ix-pool-configuration-checker',
+  templateUrl: './pool-configuration-checker.component.html',
+  styleUrls: ['./pool-configuration-checker.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class PoolConfigurationCheckerComponent implements AfterViewInit {
```

Comment:
Why not just a service with some method that would get called onInit inside of available and installed apps?